src/main/java/com/microsoft/azure/datalake/store/Core.java

```diff
@@ -10,6 +10,11 @@
 import com.fasterxml.jackson.core.JsonFactory;
 import com.fasterxml.jackson.core.JsonParser;
 import com.fasterxml.jackson.core.JsonToken;
+import com.fasterxml.jackson.databind.JsonNode;
```

Comment:
can we stick with Jackson.core only, and not bring in databind? We have had dependency version problems before with databind - so we avoid expanding the set of dependencies unless absolutely necessary.